### PR TITLE
Implement basic fastai tabular regression model.

### PR DESCRIPTION
**Pull request by erwald**
It makes predictions! Though bad ones, 'tis true.

### CODE REVIEW

**Comment by erwald** [comment]
I was in a bit of a rush when I wrote that, hence the laconic message, but of course next week I'll go through everything I've done so that no one gets left behind. Or you can come ask me about it and I'll explain everything!

In short, what it does is creates a data bunch (a PyTorch thing for serving data to a model) and a learner (basically a model with some additional functionality, as I understand it), and then I figure out a good learning rate and then train the model.

**Comment by boris-tschirschwitz** [comment]
Looks like one doesn't just review a notebook PR via diffs :-)

**Comment by erwald** [comment]
No ... one more reason not to like them. I guess you just have to check out the branch and have a look for yourself!

**Comment by boris-tschirschwitz** [comment]
You need to add faster and sklearn to requirements.txt:

fastai
kaggle
jupyter
pandas
numpy
seaborn
sklearn

File: elo_loyalty_prediction.ipynb [hunk 1]
```diff
@@ -154,15 +184,211 @@
...
+    "learn.recorder.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train Model\n",
+    "\n",
+    "Finally we train the model, with weight decay to encourage the, and then show some results."
```
Comment (boris-tschirschwitz) [inline elo_loyalty_prediction.ipynb]: to encourage the "what",

**Comment by pabloper** [comment]
Thanks for the PR Erich, I am trying to go through what you did, so many new tools for me 😸

File: elo_loyalty_prediction.ipynb [hunk 2]
```diff
@@ -181,7 +407,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "version": "3.7.1"
```
Comment (pabloper) [inline elo_loyalty_prediction.ipynb]: Why this change?

Comment (erwald) [reply elo_loyalty_prediction.ipynb]: Whoops, that was not on purpose – I suppose it did it automatically since I only have v3.7.1 installed on this computer.

@boris-tschirschwitz Whoops, seems that I once again pretended to forget to add the requirements. ;-)

I have now upgraded to v3.7.2 here, added the missing requirements and completed the half-finished subordinate clause (in addition to making some other revisions of the explanatory texts).
